haskellPackages: update stackage and hackage

maralorn opened the pull request:
### This Merge

This PR is the regular merge of the `haskell-updates` branch into `master`.

This branch is being continually built and tested by hydra at https://hydra.nixos.org/jobset/nixpkgs/haskell-updates. You may be able to find an up-to-date Hydra build report at [cdepillabout/nix-haskell-updates-status](https://github.com/cdepillabout/nix-haskell-updates-status).

We roughly aim to merge these `haskell-updates` PRs at least once every two weeks. See the @NixOS/haskell [team calendar](https://cloud.maralorn.de/apps/calendar/p/Mw5WLnzsP7fC4Zky) for who is currently in charge of this branch.

### haskellPackages Workflow Summary

Our workflow is currently described in [`pkgs/development/haskell-modules/HACKING.md`](https://github.com/NixOS/nixpkgs/blob/haskell-updates/pkgs/development/haskell-modules/HACKING.md).

The short version is this:
* We regularly update the Stackage and Hackage pins on `haskell-updates` (normally at the beginning of a merge window).
* The community fixes builds of Haskell packages on that branch.
* We aim at at least one merge of `haskell-updates` into `master` every two weeks.
* We only do the merge if the [`mergeable`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/mergeable) job is succeeding on hydra.
* If a [`maintained`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/maintained) package is still broken at the time of merge, we will only merge if the maintainer has been pinged 7 days in advance. (If you care about a Haskell package, become a maintainer!)

---

This is the follow-up to #148075. Come to [#haskell:nixos.org](https://matrix.to/#/#haskell:nixos.org) if you have any questions.

maralorn committed:
haskellPackages: stackage-lts 18.17 -> 18.18

This commit has been generated by maintainers/scripts/haskell/update-stackage.sh

maralorn committed:
all-cabal-hashes: 2021-11-18T20:32:52Z -> 2021-12-02T21:05:02Z

This commit has been generated by maintainers/scripts/haskell/update-hackage.sh

maralorn committed:
haskellPackages: regenerate package set based on current config

This commit has been generated by maintainers/scripts/haskell/regenerate-hackage-packages.sh

github-actions[bot] committed:
Merge master into haskell-updates

sternenseemann committed:
haskell.packages.ghc901.extras: drop now unnecessary override

We upgraded th-extras to a git revision before for 9.0.1 compat, but
it has now gotten a proper release.

sternenseemann committed:
haskellPackages.paramtree: update comment for dontCheck override

sternenseemann committed:
nix-tree: build with latest brick

sternenseemann committed:
haskellPackages.brick_0_64_2: preserve for matterhorn

Sadly, matterhorn can't be built with 0.65 (yet?).

sternenseemann committed:
haskell.packages.ghc921.hashable: reflect hackage update

sternenseemann committed:
haskellPackages.ghcup: reflect streamly hackage update

sternenseemann committed:
haskellPackages.hadolint: reflect dependency hackage updates

sternenseemann committed:
haskellPackages.hledger_1_24: init at 1.24

haskellPackages.hledger-lib_1_24: init at 1.24
haskellPackages.hledger_1_23: remove at 1.23
haskellPackages.hledger-lib_1_23: remove at 1.23

sternenseemann committed:
haskellPackages.graphql-engine: adjust to hspec hackage update

sternenseemann committed:
hledger-check-fancyassertions: use hledger-lib 1.24

sternenseemann committed:
haskell.packages.ghc901.path: pin to 0.9.0

This is the last version which builds with the stackage version of
genvalidity and is also compatible with 9.0.1.

sternenseemann committed:
haskell.packages.ghc901.ormolu: pin to 0.3.*

ormolu >= 0.4 seems to require GHC 9.2.1

sternenseemann committed:
haskell.packages.ghc901: adjust to dhall hackage update

sternenseemann committed:
haskell.packages.ghc921.quickcheck-instances: 0.3.26.1 -> 0.3.27

sternenseemann committed:
haskell.packages.ghc901.weeder: jailbreak to allow lens 5.1

sternenseemann committed:
haskell.packages.ghc921.ormolu: 0.1.4.1 -> 0.4.0.0

Brings GHC 9.2.1 support, also fix builds of dependencies.

github-actions[bot] committed:
Merge master into haskell-updates

sternenseemann committed:
haskellPackages.git-annex: update sha256 for 8.20211123

sternenseemann commented:
### [haskell-updates build report from hydra](https://hydra.nixos.org/jobset/nixpkgs/haskell-updates)
*evaluation [1727704](https://hydra.nixos.org/eval/1727704) of nixpkgs commit [d6bc37d](https://github.com/NixOS/nixpkgs/commits/d6bc37dcec6ead1da9e0de492fd31315c85af9c3) as of 2021-12-04 12:18 UTC*

:yellow_circle: **Potential issues**
  * `mergeable` jobset is not finished.
  * `maintained` jobset is not finished.

#### Build summary

 | Platform | Failed :x: | DependencyFailed :heavy_exclamation_mark: | Unfinished :hourglass_flowing_sand: | Success :heavy_check_mark: | 
 | --- | --- | --- | --- | --- | 
 | [aarch64-linux :iphone:](https://hydra.nixos.org/eval/1727704?filter=.aarch64-linux) | 25 | 13 | 23 | 7090 | 
 | [x86_64-darwin :apple:](https://hydra.nixos.org/eval/1727704?filter=.x86_64-darwin) | 40 | 28 | 993 | 6054 | 
 | [x86_64-linux :penguin:](https://hydra.nixos.org/eval/1727704?filter=.x86_64-linux) | 7 | 3 | 13 | 7165 | 
#### Maintained packages with build failure
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160442639) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/160452858) [[:penguin::x:]](https://hydra.nixos.org/build/160447267) [haskellPackages.dhall-nixpkgs](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.dhall-nixpkgs) @Gabriel439
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160444711) [[:apple::x:]](https://hydra.nixos.org/build/160444980) [[:penguin::x:]](https://hydra.nixos.org/build/160449346) [haskellPackages.rel8](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.rel8) @sternenseemann
#### Maintained packages with failed dependency
- [x] [git-annex](https://hydra.nixos.org/eval/1727704?filter=git-annex) 
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160438866) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160455285) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160444115) [toplevel](https://hydra.nixos.org/eval/1727704?filter=git-annex)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160439618) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160444173) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160453727) [haskellPackages](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.git-annex)
- [ ] [haskell-language-server](https://hydra.nixos.org/eval/1727704?filter=haskell-language-server) @maralorn
  - [[:iphone::hourglass_flowing_sand:]](https://hydra.nixos.org/build/160758061) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/160758067) [[:penguin::hourglass_flowing_sand:]](https://hydra.nixos.org/build/160758064) [toplevel](https://hydra.nixos.org/eval/1727704?filter=haskell-language-server)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160443318) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160445683) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160455435) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1727704?filter=haskell.packages.ghc8107.haskell-language-server)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160451533) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160441483) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160449177) [haskell.packages.ghc884](https://hydra.nixos.org/eval/1727704?filter=haskell.packages.ghc884.haskell-language-server)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160758056) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/160758066) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160758062) [haskell.packages.ghc901](https://hydra.nixos.org/eval/1727704?filter=haskell.packages.ghc901.haskell-language-server)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160446529) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160445708) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160442682) [haskellPackages](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.haskell-language-server)
- [ ] [hsdns](https://hydra.nixos.org/eval/1727704?filter=hsdns)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160445928) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160447151) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160452154) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1727704?filter=haskell.packages.ghc8107.hsdns)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160444002) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160445446) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160448296) [haskell.packages.ghc884](https://hydra.nixos.org/eval/1727704?filter=haskell.packages.ghc884.hsdns)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160457080) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/160453520) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160440975) [haskell.packages.ghc901](https://hydra.nixos.org/eval/1727704?filter=haskell.packages.ghc901.hsdns)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160450041) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160444127) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160444167) [haskell.packages.ghc921](https://hydra.nixos.org/eval/1727704?filter=haskell.packages.ghc921.hsdns)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160453268) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160452760) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160436298) [haskellPackages](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.hsdns)
- [ ] [titlecase](https://hydra.nixos.org/eval/1727704?filter=titlecase) 
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160437766) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160436845) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160453740) [haskell.packages.ghc8107](https://hydra.nixos.org/eval/1727704?filter=haskell.packages.ghc8107.titlecase)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160448969) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160446339) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160439450) [haskell.packages.ghc884](https://hydra.nixos.org/eval/1727704?filter=haskell.packages.ghc884.titlecase)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160452446) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160439364) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160446378) [haskell.packages.ghc901](https://hydra.nixos.org/eval/1727704?filter=haskell.packages.ghc901.titlecase)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160453934) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160437434) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160445155) [haskell.packages.ghc921](https://hydra.nixos.org/eval/1727704?filter=haskell.packages.ghc921.titlecase)
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160449589) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160435610) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160449551) [haskellPackages](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.titlecase)
#### Unmaintained packages with build failure
<details><summary>60 job(s) </summary>

- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160451695) [[:apple::x:]](https://hydra.nixos.org/build/160448405) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160447370) [haskellPackages.sdp](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.sdp)  :arrow_heading_up: 9 | 9
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160444791) [[:apple::x:]](https://hydra.nixos.org/build/160439292) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160437498) [haskellPackages.thyme](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.thyme)  :arrow_heading_up: 6 | 15
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160452801) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160439013) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160436279) [haskellPackages.libBF](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.libBF)  :arrow_heading_up: 4 | 20
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160452785) [[:apple::x:]](https://hydra.nixos.org/build/160456899) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160436654) [haskellPackages.exinst](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.exinst)  :arrow_heading_up: 4 | 6
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160455913) [[:apple::x:]](https://hydra.nixos.org/build/160442534) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160453493) [haskellPackages.nri-observability](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.nri-observability)  :arrow_heading_up: 3 | 5
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160455616) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160449645) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160439430) [haskellPackages.ptr-poker](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.ptr-poker)  :arrow_heading_up: 3 | 4
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160451781) [[:apple::x:]](https://hydra.nixos.org/build/160436114) [[:penguin::x:]](https://hydra.nixos.org/build/160449097) [haskellPackages.brittany](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.brittany)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160457155) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160447471) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160453021) [haskellPackages.OrderedBits](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.OrderedBits)  :arrow_heading_up: 1 | 36
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160437392) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160455656) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160447705) [haskellPackages.type-natural](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.type-natural)  :arrow_heading_up: 1 | 4
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160448126) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160451711) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160444202) [haskellPackages.long-double](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.long-double)  :arrow_heading_up: 1 | 2
- [ ] [[:iphone::hourglass_flowing_sand:]](https://hydra.nixos.org/build/160442462) [[:apple::x:]](https://hydra.nixos.org/build/160443828) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160443528) [haskellPackages.easytensor](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.easytensor)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160448163) [[:apple::x:]](https://hydra.nixos.org/build/160437057) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160452899) [haskellPackages.keep-alive](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.keep-alive)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160453043) [[:apple::x:]](https://hydra.nixos.org/build/160449027) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160446913) [haskellPackages.loc](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.loc)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160453257) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160447564) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160451962) [haskellPackages.nlopt-haskell](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.nlopt-haskell)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160454504) [[:apple::x:]](https://hydra.nixos.org/build/160443398) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160436651) [haskellPackages.opencv](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.opencv)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160448013) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160438602) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160449591) [haskellPackages.unicode-properties](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.unicode-properties)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160442826) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/160451470) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160440611) [haskellPackages.accelerate-llvm](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.accelerate-llvm)  :arrow_heading_up: 0 | 8
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160437233) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160455941) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160449533) [haskellPackages.freetype2](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.freetype2)  :arrow_heading_up: 0 | 7
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160452587) [[:apple::x:]](https://hydra.nixos.org/build/160439015) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160441762) [haskellPackages.pipes-zlib](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.pipes-zlib)  :arrow_heading_up: 0 | 6
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160450166) [[:apple::x:]](https://hydra.nixos.org/build/160442283) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160453763) [haskellPackages.hmidi](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.hmidi)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160450300) [[:apple::x:]](https://hydra.nixos.org/build/160437505) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160446616) [haskellPackages.zip](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.zip)  :arrow_heading_up: 0 | 4
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160454095) [[:apple::x:]](https://hydra.nixos.org/build/160446886) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160437771) [haskellPackages.caster](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.caster)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160448165) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/160453018) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160436100) [haskellPackages.cdar-mBound](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.cdar-mBound)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160442198) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/160451934) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160435617) [haskellPackages.quic](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.quic)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160456686) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160440055) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160447009) [haskellPackages.picosat](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.picosat)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160456993) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160445303) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160444768) [haskellPackages.HsASA](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.HsASA) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160452345) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160447544) [[:penguin::x:]](https://hydra.nixos.org/build/160436713) [haskellPackages.binary-io](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.binary-io) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160438632) [[:apple::x:]](https://hydra.nixos.org/build/160447656) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160454633) [haskellPackages.chiphunk](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.chiphunk) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160447613) [[:apple::x:]](https://hydra.nixos.org/build/160436808) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160452859) [haskellPackages.discount](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.discount) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160444436) [[:apple::x:]](https://hydra.nixos.org/build/160448564) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160450470) [haskellPackages.diskhash](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.diskhash) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160441752) [[:apple::x:]](https://hydra.nixos.org/build/160447751) [[:penguin::x:]](https://hydra.nixos.org/build/160444164) [haskellPackages.doctest-parallel](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.doctest-parallel) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160441221) [[:apple::x:]](https://hydra.nixos.org/build/160440137) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160448867) [haskellPackages.epub-tools](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.epub-tools) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160438289) [[:apple::x:]](https://hydra.nixos.org/build/160448937) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160438105) [haskellPackages.float128](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.float128) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160439808) [[:apple::x:]](https://hydra.nixos.org/build/160445636) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160443664) [haskellPackages.gerrit](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.gerrit) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160440618) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160445846) [haskellPackages.gnome-keyring](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.gnome-keyring) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160449112) [[:apple::x:]](https://hydra.nixos.org/build/160449979) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160450713) [haskellPackages.gtk-traymanager](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.gtk-traymanager) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160439838) [[:apple::x:]](https://hydra.nixos.org/build/160441480) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160444226) [haskellPackages.hid](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.hid) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160453957) [[:apple::x:]](https://hydra.nixos.org/build/160445139) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160449943) [haskellPackages.highlight](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.highlight) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160441619) [[:apple::x:]](https://hydra.nixos.org/build/160444541) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160440624) [haskellPackages.hinotify-conduit](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.hinotify-conduit) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160451239) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160440971) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160443365) [haskellPackages.hq](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.hq) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160450547) [[:apple::x:]](https://hydra.nixos.org/build/160444061) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160447603) [haskellPackages.hsshellscript](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.hsshellscript) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160451772) [[:apple::x:]](https://hydra.nixos.org/build/160442259) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160443901) [haskellPackages.hssourceinfo](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.hssourceinfo) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160449594) [[:apple::x:]](https://hydra.nixos.org/build/160440357) [[:penguin::x:]](https://hydra.nixos.org/build/160439922) [haskellPackages.jet](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.jet) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160454018) [[:apple::x:]](https://hydra.nixos.org/build/160437642) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160440164) [haskellPackages.linux-framebuffer](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.linux-framebuffer) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160453936) [[:apple::x:]](https://hydra.nixos.org/build/160437185) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160445738) [haskellPackages.mediawiki2latex](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.mediawiki2latex) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160438718) [[:apple::x:]](https://hydra.nixos.org/build/160445709) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160441864) [haskellPackages.mercury-api](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.mercury-api) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160443304) [[:apple::x:]](https://hydra.nixos.org/build/160440381) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160440929) [haskellPackages.nano-cryptr](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.nano-cryptr) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160445429) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/160456171) [[:penguin::x:]](https://hydra.nixos.org/build/160446239) [haskellPackages.pandoc-lua-marshal](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.pandoc-lua-marshal) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160455022) [[:apple::x:]](https://hydra.nixos.org/build/160441458) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160444285) [haskellPackages.ping-wrapper](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.ping-wrapper) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160450731) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160437435) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160437024) [haskellPackages.poker](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.poker) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160442276) [[:apple::x:]](https://hydra.nixos.org/build/160446379) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160451067) [haskellPackages.posix-timer](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.posix-timer) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160443661) [[:apple::x:]](https://hydra.nixos.org/build/160444992) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160440496) [haskellPackages.procex](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.procex) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160444963) [[:apple::x:]](https://hydra.nixos.org/build/160439713) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160443288) [haskellPackages.pthread](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.pthread) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160447596) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160447064) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160447489) [haskellPackages.risc386](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.risc386) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160440396) [[:apple::x:]](https://hydra.nixos.org/build/160442649) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160442404) [haskellPackages.shared-memory](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.shared-memory) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160455056) [[:apple::x:]](https://hydra.nixos.org/build/160449657) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160437421) [haskellPackages.tailfile-hinotify](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.tailfile-hinotify) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160440331) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160446697) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160445012) [haskellPackages.wiringPi](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.wiringPi) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/160446040) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160441343) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160435780) [haskellPackages.x86-64bit](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.x86-64bit) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160450612) [[:apple::x:]](https://hydra.nixos.org/build/160449704) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160441783) [haskellPackages.xmonad-utils](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.xmonad-utils) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160439908) [[:apple::x:]](https://hydra.nixos.org/build/160436448) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160453468) [haskellPackages.zxcvbn-c](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.zxcvbn-c) 
</details>

#### Unmaintained packages with failed dependency
<details><summary>34 job(s) </summary>

- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160436901) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160437166) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160443953) [haskellPackages.jsonifier](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.jsonifier)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160453873) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160450312) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160453718) [haskellPackages.sdp-io](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.sdp-io)  :arrow_heading_up: 2 | 2
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160457142) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160448997) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160453826) [haskellPackages.hls-brittany-plugin](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.hls-brittany-plugin)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160448671) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160445609) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160455676) [haskellPackages.nri-redis](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.nri-redis)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160435684) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160449924) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160442718) [haskellPackages.opentelemetry-extra](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.opentelemetry-extra)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160449600) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160441477) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160448101) [haskellPackages.orgmode-parse](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.orgmode-parse)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160450576) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160451169) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160444440) [haskellPackages.sdp-hashable](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.sdp-hashable)  :arrow_heading_up: 1 | 1
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160443027) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160444316) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160449281) [haskellPackages.PrimitiveArray](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.PrimitiveArray)  :arrow_heading_up: 0 | 35
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160457159) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160437715) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160448497) [haskellPackages.sized](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.sized)  :arrow_heading_up: 0 | 2
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160455271) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160450031) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160443599) [haskellPackages.keenser](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.keenser)  :arrow_heading_up: 0 | 1
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160454046) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160442176) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160437942) [haskellPackages.antiope-es](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.antiope-es) 
- [ ] [[:iphone::hourglass_flowing_sand:]](https://hydra.nixos.org/build/160450414) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160442758) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160438574) [haskellPackages.easytensor-vulkan](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.easytensor-vulkan) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160438508) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160445563) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160452505) [haskellPackages.exinst-aeson](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.exinst-aeson) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160444465) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160452193) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160449724) [haskellPackages.exinst-bytes](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.exinst-bytes) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160445351) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160440987) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160456270) [haskellPackages.exinst-cereal](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.exinst-cereal) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160444822) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160446684) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160436598) [haskellPackages.exinst-serialise](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.exinst-serialise) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160440939) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160453008) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160438885) [haskellPackages.fastparser](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.fastparser) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160457185) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160440420) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160443991) [haskellPackages.hmatrix-nlopt](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.hmatrix-nlopt) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160439447) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160450831) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160446334) [haskellPackages.nri-http](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.nri-http) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160456306) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160435940) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160452954) [haskellPackages.nri-test-encoding](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.nri-test-encoding) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160447547) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160448440) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160443745) [haskellPackages.opencv-extra](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.opencv-extra) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160452923) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160446545) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160438494) [haskellPackages.opentelemetry-lightstep](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.opentelemetry-lightstep) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160447300) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160447408) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160447619) [haskellPackages.orgstat](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.orgstat) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160452388) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160455535) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160448627) [haskellPackages.postgresql-replicant](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.postgresql-replicant) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160455264) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160445032) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160444910) [haskellPackages.rounded](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.rounded) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160450463) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160456073) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160455034) [haskellPackages.scan-metadata](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.scan-metadata) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160440397) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160450917) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160447967) [haskellPackages.sdp-binary](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.sdp-binary) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160452049) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160452947) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160443606) [haskellPackages.sdp-deepseq](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.sdp-deepseq) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160447665) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160449335) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160456469) [haskellPackages.sdp-quickcheck](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.sdp-quickcheck) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160451505) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160438040) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160453349) [haskellPackages.sdp4bytestring](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.sdp4bytestring) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160448312) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160453383) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160449890) [haskellPackages.sdp4text](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.sdp4text) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160437843) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160438305) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160447385) [haskellPackages.sdp4unordered](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.sdp4unordered) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/160435692) [[:apple::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160444268) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160447767) [haskellPackages.sdp4vector](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.sdp4vector) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/160451788) [[:apple::heavy_check_mark:]](https://hydra.nixos.org/build/160453694) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/160439350) [haskellPackages.unicode-names](https://hydra.nixos.org/eval/1727704?filter=haskellPackages.unicode-names) 
</details>

#### Top 50 broken packages, sorted by number of reverse dependencies
<details><summary>50 job(s) </summary>

[haskell98](https://packdeps.haskellers.com/reverse/haskell98) :arrow_heading_up: 153  
[enumerator](https://packdeps.haskellers.com/reverse/enumerator) :arrow_heading_up: 56  
[derive](https://packdeps.haskellers.com/reverse/derive) :arrow_heading_up: 48  
[contiguous](https://packdeps.haskellers.com/reverse/contiguous) :arrow_heading_up: 46  
[MonadCatchIO-transformers](https://packdeps.haskellers.com/reverse/MonadCatchIO-transformers) :arrow_heading_up: 41  
[parseargs](https://packdeps.haskellers.com/reverse/parseargs) :arrow_heading_up: 41  
[bytesmith](https://packdeps.haskellers.com/reverse/bytesmith) :arrow_heading_up: 36  
[data-lens](https://packdeps.haskellers.com/reverse/data-lens) :arrow_heading_up: 33  
[distributed-process](https://packdeps.haskellers.com/reverse/distributed-process) :arrow_heading_up: 30  
[iteratee](https://packdeps.haskellers.com/reverse/iteratee) :arrow_heading_up: 29  
[jmacro](https://packdeps.haskellers.com/reverse/jmacro) :arrow_heading_up: 29  
[ip](https://packdeps.haskellers.com/reverse/ip) :arrow_heading_up: 26  
[either-unwrap](https://packdeps.haskellers.com/reverse/either-unwrap) :arrow_heading_up: 25  
[HList](https://packdeps.haskellers.com/reverse/HList) :arrow_heading_up: 23  
[SciBaseTypes](https://packdeps.haskellers.com/reverse/SciBaseTypes) :arrow_heading_up: 22  
[haskelldb](https://packdeps.haskellers.com/reverse/haskelldb) :arrow_heading_up: 22  
[hsc3](https://packdeps.haskellers.com/reverse/hsc3) :arrow_heading_up: 22  
[wxdirect](https://packdeps.haskellers.com/reverse/wxdirect) :arrow_heading_up: 22  
[BiobaseTypes](https://packdeps.haskellers.com/reverse/BiobaseTypes) :arrow_heading_up: 21  
[wxc](https://packdeps.haskellers.com/reverse/wxc) :arrow_heading_up: 21  
[biocore](https://packdeps.haskellers.com/reverse/biocore) :arrow_heading_up: 20  
[polysemy-plugin](https://packdeps.haskellers.com/reverse/polysemy-plugin) :arrow_heading_up: 20  
[secp256k1-haskell](https://packdeps.haskellers.com/reverse/secp256k1-haskell) :arrow_heading_up: 20  
[wxcore](https://packdeps.haskellers.com/reverse/wxcore) :arrow_heading_up: 20  
[attoparsec-enumerator](https://packdeps.haskellers.com/reverse/attoparsec-enumerator) :arrow_heading_up: 19  
[bytestring-show](https://packdeps.haskellers.com/reverse/bytestring-show) :arrow_heading_up: 19  
[numhask](https://packdeps.haskellers.com/reverse/numhask) :arrow_heading_up: 19  
[wx](https://packdeps.haskellers.com/reverse/wx) :arrow_heading_up: 19  
[BiobaseENA](https://packdeps.haskellers.com/reverse/BiobaseENA) :arrow_heading_up: 18  
[asn1-data](https://packdeps.haskellers.com/reverse/asn1-data) :arrow_heading_up: 18  
[dbus-core](https://packdeps.haskellers.com/reverse/dbus-core) :arrow_heading_up: 18  
[gtksourceview2](https://packdeps.haskellers.com/reverse/gtksourceview2) :arrow_heading_up: 18  
[BiobaseXNA](https://packdeps.haskellers.com/reverse/BiobaseXNA) :arrow_heading_up: 17  
[HGamer3D-Data](https://packdeps.haskellers.com/reverse/HGamer3D-Data) :arrow_heading_up: 17  
[certificate](https://packdeps.haskellers.com/reverse/certificate) :arrow_heading_up: 17  
[dbus-client](https://packdeps.haskellers.com/reverse/dbus-client) :arrow_heading_up: 17  
[gconf](https://packdeps.haskellers.com/reverse/gconf) :arrow_heading_up: 17  
[gtk-serialized-event](https://packdeps.haskellers.com/reverse/gtk-serialized-event) :arrow_heading_up: 17  
[uuid-orphans](https://packdeps.haskellers.com/reverse/uuid-orphans) :arrow_heading_up: 17  
[cuda](https://packdeps.haskellers.com/reverse/cuda) :arrow_heading_up: 16  
[happstack-jmacro](https://packdeps.haskellers.com/reverse/happstack-jmacro) :arrow_heading_up: 16  
[manatee-core](https://packdeps.haskellers.com/reverse/manatee-core) :arrow_heading_up: 16  
[monads-fd](https://packdeps.haskellers.com/reverse/monads-fd) :arrow_heading_up: 16  
[murmur3](https://packdeps.haskellers.com/reverse/murmur3) :arrow_heading_up: 16  
[tls-extra](https://packdeps.haskellers.com/reverse/tls-extra) :arrow_heading_up: 16  
[ADPfusion](https://packdeps.haskellers.com/reverse/ADPfusion) :arrow_heading_up: 15  
[MaybeT](https://packdeps.haskellers.com/reverse/MaybeT) :arrow_heading_up: 15  
[blaze-builder-enumerator](https://packdeps.haskellers.com/reverse/blaze-builder-enumerator) :arrow_heading_up: 15  
[clash-prelude](https://packdeps.haskellers.com/reverse/clash-prelude) :arrow_heading_up: 15  
[hetero-dict](https://packdeps.haskellers.com/reverse/hetero-dict) :arrow_heading_up: 15  
</details>


*:arrow_heading_up:: The number of packages that depend (directly or indirectly) on this package (if any). If two numbers are shown the first (lower) number considers only packages which currently have enabled hydra jobs, i.e. are not marked broken. The second (higher) number considers all packages.*

*Report generated with [maintainers/scripts/haskell/hydra-report.hs](https://github.com/NixOS/nixpkgs/blob/haskell-updates/maintainers/scripts/haskell/hydra-report.sh)*


----------------------------------------------------------------------

This README.md is automatically updated every 6 hours with the status of the
[`haskell-updates` branch/jobset on Hydra](https://hydra.nixos.org/jobset/nixpkgs/haskell-updates)
from [Nixpkgs](https://github.com/NixOS/nixpkgs).  This is mostly only of
interest to the [Nixpkgs Haskell maintainers](https://github.com/orgs/NixOS/teams/haskell).

See the
[haskell-modules/HACKING.md](https://github.com/NixOS/nixpkgs/blob/haskell-updates/pkgs/development/haskell-modules/HACKING.md)
file for more information about this build report.

You can force the GitHub Action to run (and the README.md to be updated) by
manually running the Action.  To do this, go to the Action list screen
(https://github.com/cdepillabout/nix-haskell-updates-status/actions),
click on any of the Workflow runs, and then click the `Re-run jobs` button.

sternenseemann committed:
haskellPackages.rel8: downgrade to 1.2.0.0 for stackage LTS compat

github-actions[bot] committed:
Merge master into haskell-updates

Gabriel439 commented:
I revised `dhall-nixpkgs-1.0.7` to lower the bound on `base64-bytestring` so it will work against the default version.  Once the revision is picked up then that should fix the `dhall-nixpkgs` build failure

sternenseemann committed:
haskellPackages.dhall-nixpkgs: use revised cabal file

This allows the Stackage LTS version of base64-bytestring (1.1.0.0).

sternenseemann committed:
haskellPackages.dhall-nixpkgs: generate shell completions

sternenseemann committed:
Merge pull request #148699 from sternenseemann/dhall-nixpkgs-fix

haskellPackages.dhall-nixpkgs: build with base64-bytestring 1.1.0.0

sternenseemann commented:
@Gabriel439 Thanks, reflected in 5606343aa538da916fa6c6f03837544b218ee624!

sternenseemann committed:
haskellPackages.brittany: pin to 0.13.1.2

0.14.0.0 introduces support for GHC 9.0.1, but also drops support for
all GHCs below, so we can't upgrade to that version.

For the 9.0.1 hls brittany support is now possible in theory. In
practice however, it is a massive pain to get to work, as britanny
depends on the latest and greatest version of multiple packages that are
pinned by Stackage LTS.

sternenseemann committed:
haskell.packages.ghc901.{multistate,butcher}: jailbreak

These packages build fine with GHC 9.0.1, but forbid base 4.15. Build
fixed as a prerequisite to later unbreaking brittany.

sternenseemann committed:
haskell.packages.ghc901.text-short: 0.1.3 -> 0.1.4

With GHC 9.0.1 text-short's test suite hits a bug that was fixed in 0.1.4.

sternenseemann committed:
haskellPackages.photoname: disable test suite requiring stack

sternenseemann committed:
haskellPackages: mark builds failing on hydra as broken

This commit has been generated by maintainers/scripts/haskell/mark-broken.sh

sternenseemann committed:
haskellPackages.jet: allow building with recursive-zipper 0.0.0.1

sternenseemann committed:
Merge remote-tracking branch 'origin/master' into haskell-updates

sternenseemann committed:
haskellPackages.proto-lens: drop now unnecessary override

github-actions[bot] committed:
Merge master into haskell-updates

sternenseemann committed:
haskell.packages.ghc921.memory: ensure head.hackage patch applies

sternenseemann committed:
Merge remote-tracking branch 'origin/master' into haskell-updates
